statuspage.js

```diff
@@ -147,15 +147,15 @@ define(function(require, exports, module) {
                     // show banner for resolved incidents
                     Object.keys(unresolvedIncidents).forEach(function(id) {
```

Comment:
@kzidane, exactly what will the user see if there are multiple incidents? One banner at a time for n seconds?